recipes/libharu/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "libharu" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libharu/libharu/archive/RELEASE_{{ version.replace('.', '_') }}.tar.gz
+  sha256: 8f9e68cc5d5f7d53d1bc61a1ed876add1faf4f91070dbc360d8b259f46d9a4d2
+  patches:
+    - 0001-skip-installing-if-directory.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+  host:
+    - zlib
+    - libpng
+    - boost
+  run:
+    - zlib
+    - libpng
+    - boost
+
+test:
+  commands:
+    - test -f $PREFIX/include/hpdf.h  # [unix]
+    - test -f $PREFIX/lib/libhpdf${SHLIB_EXT}  # [unix]
+    - if not exist %LIBRARY_INC%\hpdf.h exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\libhpdf.dll exit 1  # [win]
+
+about:
+  home: http://libharu.org/
+  license: Zlib
+  license_family: OTHER
+  license_file: LICENSE.txt
+  summary: Simple, fast, extensible JSON encoder/decoder for Python
```

Comment:
You need to update the description.

Reply:
Oops!  What a silly oversight on my part.  I have moved up the description, which is short enough to be a summary.